apps/dialtone-documentation/docs/components/dropdown.md

```diff
@@ -34,13 +41,150 @@ figma_url: https://www.figma.com/file/2adf7JhZOncRyjYiy2joil/DT-Core%3A-Componen
         heading-class="d-py4 d-px8 d-fw-semibold d-c-default"
         heading="Menu Heading B"
       >
-        <dt-list-item>
+        <dt-list-item
+          role="menuitem"
+          navigation-type="arrow-keys"
+          @click="close"
+        >
+          Menu Item 3
+        </dt-list-item>
+      </dt-list-item-group>
+    </template>
+  </dt-dropdown>
+</code-well-header>
+
+## Usage
+
+Use the Dropdown component when you have a list of links or actions that can be performed. Use the [Select Menu](/components/select-menu.html) component to allow the users to choose one option from the list.
+
+The dropdown component provides accessibility controls when using with List item component.
+
+The dropdown has 2 major pieces that are provided as required named slots: the anchor and the list.
+
+The **anchor** slot contains the element that controls the dropdown's visibility.
+
+The **list** slot contains a list of items to show as dropdown menu items.
+
+## Variants and examples
+
+### Default
+
+<code-well-header>
+  <dt-dropdown navigation-type="arrow-keys">
+    <template #anchor="{ attrs }">
+      <dt-button v-bind="attrs">
+        Click to open
+      </dt-button>
+    </template>
+    <template #list="{ close }">
+      <dt-list-item
+        v-for="(item) in items"
+        :key="item.id"
+        role="menuitem"
+        :navigation-type="arrow-keys"
+        @click="close"
+      >
+        {{ item.name }}
+      </dt-list-item>
+    </template>
+  </dt-dropdown>
+</code-well-header>
+
+<code-example-tabs
+htmlCode='
+<div>
+  <div class="d-popover">
+    <div id="DtPopover__anchor2">
+      <button class="base-button__button d-btn d-btn--primary">
+        <span class="d-btn__label base-button__label"> Click to open </span>
+      </button>
+    </div>
+  </div>
+</div>
+<div class="tippy-box d-ps-absolute" data-tippy-root="" id="tippy-13" data-popper-placement="bottom" style="...">
+  <div id="dt7" role="menu" aria-hidden="false" aria-labelledby="DtPopover__anchor8" aria-modal="false" class="d-popover__dialog d-popover__dialog--modal" tabindex="-1" style="...">
```

Comment:
We usually prune a little bit the html code to avoid unnecessary code and have it more clear.
In this case we can remove things like role, aria-, data-